fastai/conv_learner.py

```diff
@@ -95,15 +95,20 @@ class ConvLearner(Learner):
     def __init__(self, data, models, precompute=False, **kwargs):
         self.precompute = False
         super().__init__(data, models, **kwargs)
-        if hasattr(data, 'is_multi'):
```

Comment:
This isn't always true - many users use their own custom ModelData classes. So need to put this back